protocols/ping/tests/ping.rs

```diff
@@ -266,3 +268,18 @@ impl Arbitrary for MuxerChoice {
         *g.choose(&[MuxerChoice::Mplex, MuxerChoice::Yamux]).unwrap()
     }
 }
+
+#[derive(NetworkBehaviour, Default)]
+struct Behaviour {
+    keep_alive: behaviour::KeepAlive,
+    ping: ping::Behaviour,
+}
+
+impl Behaviour {
+    fn new(config: ping::Config) -> Self {
+        Self {
+            keep_alive: behaviour::KeepAlive,
+            ping: ping::Behaviour::new(config),
```

Comment:
Hmm, I just realised that the naming here is inverted. Should it perhaps be `keep_alive::Behaviour`?

Reply:
Overall, the modules would be: `libp2p::swarm::dummy::{Behaviour,ConnectionHandler}` and possibly more. I think that would work well because it groups all "dummy" things together which is how they are likely to be used instead of grouping them by "layer", i.e. behaviour, connection handler etc.

Reply:
Perhaps we can even have a `libp2p-dummy` crate? :)

Reply:
> Perhaps we can even have a `libp2p-dummy` crate? :)

For now we could emulate this in the root crate by re-exporting `libp2p::swarm::dummy` as `libp2p::dummy`.

Reply:
I prefer `libp2p_swarm::behaviour::keep_alive::Behaviour` and `libp2p_ping::Behaviour`.

That said, I don't feel strongly about it. I mostly just care about consistency.

Reply:
Might as well do it properly then :)